app/src/docs/v1.api-spec.yaml

```diff
@@ -661,22 +730,16 @@ paths:
             type: string
           required: true
           description: a short name for the Form
-      requestBody:
-        description: A Submission object
-        required: true
-        content:
-          application/json:
-            schema:
-              type: array
-              items:
-                type: object
-                $ref: '#/components/schemas/Submission'
+        - in: path
+          name: submissionId
+          schema:
+            type: string
+            format: uuid
+          required: true
+      summary: Soft-deletes the submission with the specified ID.
       responses:
         '200':
-          description: https://ches-master-9f0fbe-prod.pathfinder.gov.bc.ca/api/v1/docs#operation/postMerge
-
-        '401':
-          $ref: '#/components/responses/UnauthorizedError'
+          description: Submission was deleted.
         '403':
           $ref: '#/components/responses/Forbidden'
         default:
```

Comment:
Double check if this needs a 404 or not - as it's a delete operation, you probably shouldn't be able to delete something that doesn't already exist or is already deleted. Granted, "delete" is a soft operation, so the record still exists - not sure if we ever came to a formal decision on how to behave when you try deleting something that already got deleted.